test/helpers/test_helpers.mocks.dart

```diff
@@ -3,51 +3,53 @@
 // Do not manually edit this file.
 
 // ignore_for_file: no_leading_underscores_for_library_prefixes
-import 'dart:async' as _i4;
-import 'dart:io' as _i17;
-import 'dart:ui' as _i9;
+import 'dart:async' as _i5;
+import 'dart:io' as _i19;
+import 'dart:ui' as _i10;
 
-import 'package:connectivity_plus/connectivity_plus.dart' as _i24;
+import 'package:connectivity_plus/connectivity_plus.dart' as _i26;
 import 'package:connectivity_plus_platform_interface/connectivity_plus_platform_interface.dart'
-    as _i25;
+    as _i27;
 import 'package:flutter/material.dart' as _i1;
 import 'package:graphql_flutter/graphql_flutter.dart' as _i3;
+import 'package:image_cropper/image_cropper.dart' as _i4;
+import 'package:image_picker/image_picker.dart' as _i13;
 import 'package:mockito/mockito.dart' as _i2;
-import 'package:qr_code_scanner/src/qr_code_scanner.dart' as _i30;
```

Comment:
Please try not changing so many things when not required.

Reply:
This file is generated by build_runner as I added mocks for two classes so I didn't change anything manually here